system/Database/Postgre/Connection.php

```diff
@@ -461,9 +461,11 @@ protected function _enableForeignKeyChecks()
      */
     public function error(): array
     {
+        $lastError = pg_last_error($this->connID);
+
         return [
             'code'    => '',
-            'message' => pg_last_error($this->connID) ?: '',
+            'message' => ! in_array($lastError, ['', '0'], true) ? $lastError : '',
```

Comment:
I don't think `pg_last_error` return type would be a falsy string, maybe just empty string? so in this case we can just remove the `?: ''`

Reply:
do you mean change to : 

```suggestion
            'message' => pg_last_error($this->connID),
```

Reply:
yes